utils/perf.c

```diff
@@ -277,6 +279,7 @@ int setup_perf_data(struct uftrace_data *handle)
 		perf[i].fp = fopen(globbuf.gl_pathv[i], "r");
 		if (perf[i].fp == NULL)
 			pr_err("open failed: %s", globbuf.gl_pathv[i]);
+		perf[i].name = xstrdup(globbuf.gl_pathv[i]);
```

Comment:
I'm not sure the ordering is correct (like perf-cpu23 vs. perf-cpu4)

Reply:
I know that `perf-cpu23` comes first than `perf-cpu4`. It's a trivial problem, but looks tricky to handle.

Reply:
Yes, as @honggyukim said earlier, it is not a big problem and is related with linux filesystem directory ordering. But I agree that it needs to be ordered properly. I think it can be the one of future work. How do you think about that?

Reply:
Hmm.. did you test it with kernel data?  I think it might not work..

Reply:
What about working on this only for `perf-cpuXX` data in this PR? It doesn't have to make it too much complicated to kernel data as of now.

Reply:
I'm ok with that.